themes/bootstrap3/templates/error/contact.phtml

```diff
@@ -0,0 +1,5 @@
+<?=$this->transEsc('Please contact the Library Reference Department for assistance')?>
```

Comment:
While we're refactoring this anyway, how would you feel about making this two separate `<p>` tags inside this template, instead of using a `<br>` here and wrapping the render in `<p>`? That seems like it might be cleaner markup, though I won't argue further if you disagree. :-)

Reply:
Good idea, now done.